library/tiqr/Tiqr/Message/FCM.php

```diff
@@ -93,7 +93,7 @@ private function _sendFirebase($deviceToken, $alert, $challenge, $apiKey)
         }
 
         if ($statusCode !== 200) {
-            throw new Tiqr_Message_Exception_SendFailure("Invalid status code : ".$statusCode, true);
+            throw new Tiqr_Message_Exception_SendFailure("Invalid status code : '".$statusCode."'. Response: ".$result, true);
```

Comment:
This will work fine, I personally like to use `sprintf` formatted strings to do this kind of reporting. Makes things more readable and easier to maintain.